src/clj/ataru/applications/excel_export.clj

```diff
@@ -140,7 +140,8 @@
   (doseq [meta-field application-meta-fields]
     (let [meta-value ((or (:format-fn meta-field) identity) ((:field meta-field) application))]
       (writer 0 (:column meta-field) meta-value)))
-  (doseq [answer (:answers application)]
+  (doseq [answer (:answers application)
+          :when (some (comp (partial = (:label answer)) :header) headers)]
```

Comment:
What will happen if we have multiple questions with the same label (which doesn't make much sense but...)?